torch/csrc/jit/codegen/cuda/transform_replay.cpp

```diff
@@ -880,7 +880,7 @@ bool validateDomain(TensorView* tv, TensorDomain* new_td) {
   auto first_mismatch =
       BestEffortReplay::findFirstMismatchedID(tv->domain(), new_td);
   return first_mismatch >= (int)tv->getMaxProducerPosition() &&
-      first_mismatch >= (int)tv->getComputeAtPosition();
+      first_mismatch >= (int)tv->getMaxComputeAtPosition();
```

Comment:
I think this should be `first_mismatch >= (int)tv->getComputeAtPosition()`? During transform propagation, the new tensor domain should neither violate the compute at position nor the compute with position, and compute at position is always stricter here.

Reply:
The compute-with position is always greater or equal to the compute-at position, so I think this is correct. We want to make sure any mismatch only happens left of the compute-at and compute-with positions.

Reply:
Oh yeah, nvm